pod/perllocale.pod

```diff
@@ -1680,12 +1722,19 @@ control, but doesn't.  If two strings do collate identically, the one
 containing the C<NUL> will sort to earlier.  Prior to 5.26, there were
 more bugs.
 
+=head2 C<LANGUAGE>
+
+As stated above, Perl ignores this environment variable.
+
 =head2 Multi-threaded
 
 XS code or C-language libraries called from it that use the system
 L<C<setlocale(3)>> function (except on Windows) likely will not work
 from a multi-threaded application without changes.  See
-L<perlxs/Locale-aware XS code>.
+L<perlxs/Locale-aware XS code>.  Some such have been so changed, but
+call C libraries that can't be changed.  This will work, but as long as
```

Comment:
Wording of the two lines above feels awkward, *e.g.,* "Some such ..." *what?*

Reply:
I took this opportunity to make a bunch more changes.